test-snapshots/snapshots-output/Snapshot.Literals.ss

```diff
@@ -14,8 +14,8 @@
     record2
     string
   )
-  (imports
-    (prefix (library (chezscheme)) scm:)
+  (import
+    (prefix ((chezscheme)) scm:)
```

Comment:
Latest commit: `imports` was a typo.

Also, after re-reading that part in the R6RS spec, `library` is only needed if the first identifier is one of the keywords that could be confused for an import set. When I copy-pasted this file in the Scheme REPL, it rejected it until I made this change.